addons/binding/org.openhab.binding.helios/ESH-INF/thing/channel-types.xml

```diff
@@ -0,0 +1,173 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="helios" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <!-- Helios Channel Types -->
+
+    <channel-type id="keypressedstamp">
+        <item-type>DateTime</item-type>
+        <label>Time a key was pressed</label>
+        <description>Event date and time in CCYY-MM-DDThh:mm:ss format</description>
+        <state readOnly="true" />
+    </channel-type>
+    <channel-type id="keypressed">
```

Comment:
Now that we have trigger channels, you might want to replace "keypressed" and "keyreleased" by a trigger channel, which simply sends key press/release events (assuming that a "state" is not really relevant here).

Reply:
Good point. They did not exist way back in 2014 ;-) But keypressed does actually send a string with the key that was pressed on the device's keyboard